src/test/java/com/google/step/finscholar/data/ScholarshipTest.java

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.data;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+
+@RunWith(JUnit4.class)
+public class ScholarshipTest {
+  private Scholarship scholarshipData1;
+  private Scholarship scholarshipData2;
+  private Scholarship customScholarship;
+
+  private static final String TEST_NAME = "Scholarship Name";
+  private static final List<UUID> TEST_SCHOOLS = List.of(UUID.randomUUID());
+  private static final UUID TEST_ID = UUID.randomUUID();
+  private static final String EMPTY_URL = "";
+   
+  @Before
+  public void setup() {
+    scholarshipData1 = ScholarshipSamples.scholarshipSampleOne;
+    scholarshipData2 = ScholarshipSamples.scholarshipSampleTwo;
+    customScholarship =  new Scholarship(TEST_NAME, TEST_ID, TEST_SCHOOLS, EMPTY_URL);
+  }
+
+  @Test
+  public void testGetName() {
```

Comment:
Test behaviors not methods. Getters probably don't need their own testing, setters can be tested by constructing one large scholarship object and making sure all the fields are set as expected. The behavior test could be "createsScholarshipForLowIncomeHispanicStudents"